new-harts-rules-the-oxford-style-guide.csl

```diff
@@ -242,9 +246,16 @@
   </macro>
   <macro name="container-title-note">
```

Comment:
Can you please put the term and container title inside a group with `delimiter=" "` like I did with the above macro? 

This will prevent "in" from being printed if there is no container title